File: server/data/results/translations/plain/java/gcd_lcm.java
public class Main {

    public static long gcd(long a, long b) {
        while (b != 0) {
            long temp = b;
            b = a % b;
            a = temp;
        }
        return a;
    }

    public static long lcm(long a, long b) {
        if (a == 0 || b == 0) {
            return 0;
        }
        return (a / gcd(a, b)) * b;
    }

    public static void main(String[] args) {
        long[][] pairs = {
            {12, 18},
            {7, 13},
            {100, 75}
        };
        for (long[] pair : pairs) {
            long a = pair[0];
            long b = pair[1];
            System.out.println(gcd(a, b) + " " + lcm(a, b));
        }
    }
}

File: server/data/results/translations/plain/go/gcd_lcm.go
package main

import (
	"fmt"
	"strconv"
)

func gcd(a, b int) int {
	for b != 0 {
		temp := b
		b = a % b
		a = temp
	}
	return a
}

func lcm(a, b int) int {
	if a == 0 || b == 0 {
		return 0
	}
	return a / gcd(a, b) * b
}

func main() {
	pairs := [][2]int{
		{12, 18},
		{7, 13},
		{100, 75},
	}
	for _, p := range pairs {
		a, b := p[0], p[1]
		fmt.Println(strconv.Itoa(gcd(a, b)) + " " + strconv.Itoa(lcm(a, b)))
	}
}